Running-Mastodon/PgBouncer-guide.md

```diff
@@ -13,13 +13,18 @@ Note that you can check "PgHero" in the administration view to see how many Post
 Installing PgBouncer
 -----
 
-On Ubuntu:
+On Ubuntu/Debian:
 
     sudo apt install pgbouncer
 
-Restarting:
+Then:
 
-    sudo service pgbouncer restart
+    nano /etc/default/pgbouncer
+    START=1
```

Comment:
I never needed to add `START=1`, not sure why, but might be worth mentioning when it's required? Just checked my own `/etc/default/pgbouncer` and it's still blank.